### PR TITLE
Kshitiz

### PR DESCRIPTION
**Pull request by kshitizdublish**

### CODE REVIEW

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts [hunk 1]
```diff
@@ -4,63 +4,75 @@ import { FloatSiteBandwidthService } from '../../../services/float-site-bandwidt
...
+			this.bandwidthDetails = resp as Array<'site_bandwidth'>;
+			Object.assign(this, { resp })
+		});
+	}
 
-  onSelect(event) {
-    console.log(event);
-  }
+	onSelect(event) {
+		console.log(event);
```
Comment (happiestRaviThapa) [inline src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: just log ? Either meaning full logs or no logs

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts [hunk 2]
```diff
@@ -4,63 +4,75 @@ import { FloatSiteBandwidthService } from '../../../services/float-site-bandwidt
...
+	}
+
+	onChangeDay(option) {
+		console.log('Selected Option = ' + option);
+	}
```
Comment (happiestRaviThapa) [inline src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: Meaning full logs or no logs. Log patterns

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts [hunk 3]
```diff
@@ -4,63 +4,75 @@ import { FloatSiteBandwidthService } from '../../../services/float-site-bandwidt
...
-  templateUrl: './float-site-bandwidth-info.component.html',
-  styleUrls: ['./float-site-bandwidth-info.component.scss']
+	selector: 'app-float-site-bandwidth-info',
+	templateUrl: './float-site-bandwidth-info.component.html',
+	styleUrls: ['./float-site-bandwidth-info.component.scss']
 })
 export class FloatSiteBandwidthInfoComponent implements OnInit {
-  private toggle: boolean = true;
-  bandwidthDetails: Array<any> = [];
+	private toggle: boolean = true;
```
Comment (happiestRaviThapa) [inline src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: toggle what ?

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts [hunk 4]
```diff
@@ -4,63 +4,75 @@ import { FloatSiteBandwidthService } from '../../../services/float-site-bandwidt
...
+	};
 
-  // line, area
-  autoScale = true;
+	// line, area
+	autoScale = true;
 
 
-  constructor(private translate: TranslateService, private _customEvents: CustomEvents, private _bandwidthService: FloatSiteBandwidthService) {
+	constructor(private translate: TranslateService, private _customEvents: CustomEvents, private _bandwidthService: FloatSiteBandwidthService) {
```
Comment (happiestRaviThapa) [inline src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: "TranslateService" is not used in component level so removed it.

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.html [hunk 5]
```diff
@@ -1,49 +1,55 @@
...
+				<div class="col-sm-8 bandwidth-chart">
+					<div class="">
+
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="bandwidth-right-col">
+			<i class="fa fa-angle-down fa-2x" aria-hidden="true" (click)="clickEvent($event)"></i>
+		</div>
```
Comment (happiestRaviThapa) [inline src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.html]: "clickEvent" what click event ?

**Comment by vijayankrishnan** [comment]
changes added as per code review

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts (hunk 2 above)
Comment (vijayankrishnan) [reply src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: changes added as per code review

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts (hunk 1 above)
Comment (vijayankrishnan) [reply src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: changes added as per code review

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts (hunk 4 above)
Comment (vijayankrishnan) [reply src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: changes added as per code review

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts (hunk 3 above)
Comment (vijayankrishnan) [reply src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.ts]: changes added as per code review

File: src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.html (hunk 5 above)
Comment (vijayankrishnan) [reply src/app/components/dashboard/float-site-bandwidth-info/float-site-bandwidth-info.component.html]: changes added as per code review

**Pull request merged**